app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,44 @@
+<h1> Welcome to your to-do list! Your tasks are: </h1>
+
+<div class = "tasklist">
+  <% @task_list.each do |task| %>
+  <% if @task_list == nil %>
```

Comment:
Oh yeah! Thanks, that is a really good note. This was my first project using ActiveRecord, and I was still pretty shaky on it so I used .where a LOT. This is why I'm looping--I need to get into the array. I'm feeling more comfortable with ActiveRecord now, so I'll be able to avoid this in the future.